distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -95,61 +113,179 @@ public void afterExam() throws Exception {
+        FTPSClient client = null;
+        try {
+            client = createSecureClient(false);
+            disconnectClient(client);
+            fail("Expected fail to complete SSL handshake between FTPS client and FTP server");
+        } catch (SSLException e) {
+            disconnectClient(client);
+        }
     }
 
     /**
      * Upload a file via insecure FTP for ingest using input stream
-     * 
+     *
      * @throws Exception
      */
     @Test
-    public void testInsecureFtpPut() throws Exception {
-        client = createInsecureClient();
+    public void testFtpPut() throws Exception {
+        setClientAuthConfiguration(CLIENT_AUTH_WANT);
+        FTPClient client = createInsecureClient();
 
-        ftpPut(SAMPLE_DATA);
+        ftpPut(client, SAMPLE_DATA);
 
         // verify FTP PUT resulted in ingest, catalogued data
         Response response = executeOpenSearch("xml", "q=*", "count=100");
-        response.then().log().all().body("metacards.metacard.size()", equalTo(1));
+        response.then()
+                .log()
+                .all()
+                .body("metacards.metacard.size()", equalTo(1));
 
         // clean up test data
         TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
     }
 
     /**
      * Upload a file via insecure FTP for ingest using streaming method
-     * 
+     *
      * @throws Exception
      */
     @Test
-    public void testInsecureFtpPutStream() throws Exception {
-        client = createInsecureClient();
+    public void testFtpPutStream() throws Exception {
+        setClientAuthConfiguration(CLIENT_AUTH_WANT);
+        FTPClient client = createInsecureClient();
 
         StringWriter writer = new StringWriter();
         IOUtils.copy(getClass().getResourceAsStream("/metacard1.xml"), writer);
-        ftpPutStreaming(writer.toString());
+        ftpPutStreaming(client, writer.toString());
 
         // verify FTP PUT resulted in ingest, catalogued data
         Response response = executeOpenSearch("xml", "q=*", "count=100");
-        response.then().log().all().body("metacards.metacard.size()", equalTo(1));
+        response.then()
+                .log()
+                .all()
+                .body("metacards.metacard.size()", equalTo(1));
 
         // clean up test data
         TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
+    }
+
+    /**
+     * Upload a file via FTPS for ingest using input stream
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFtpsPut() throws Exception {
+        setClientAuthConfiguration(CLIENT_AUTH_NEED);
+        FTPSClient client = createSecureClient(true);
+
+        ftpPut(client, SAMPLE_DATA);
+
+        // verify FTP PUT resulted in ingest, catalogued data
+        Response response = executeOpenSearch("xml", "q=*", "count=100");
+        response.then()
+                .log()
```

Comment:
We should avoid adding logging, it will just become noise in the logs while running all the tests as well as increase the already enormous size of logs.